entries/n/neopets.com.json

```diff
@@ -0,0 +1,12 @@
+{
+  "NeoPass": {
```

Comment:
```suggestion
  "Neopets": {
```

Reply:
Thanks for your correction, I misread the title of the page, it is indeed **Neopest**. 👍